### PR TITLE
use config extension and sim config

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Merge together with https://github.com/tue-robotics/tue_config/pull/16

### CODE REVIEW

File: launch/state_machines/free_mode.launch [hunk 1]
```diff
@@ -16,6 +16,7 @@
         <include file="$(find robot_launch_files)/launch/world_modeling/world_model.launch">
             <arg name="machine" value="hero2"/>
             <arg name="config" value="world_modeling/world_model.yaml"/>
+            <arg name="config_sim" value="world_modeling/world_model_sim.yaml"/>
```
Comment (jlunenburg) [inline launch/state_machines/free_mode.launch]: Why load config and sim config at the same time?

File: parameters/world_modeling/world_model_base.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+plugins:
```
Comment (jlunenburg) [inline parameters/world_modeling/world_model_base.yaml]: Improve scalability: put every plugin in a separate file. Build up world model base.yaml and include all ones there.

File: parameters/world_modeling/world_model_gmapping.yaml [hunk 3]
```diff
@@ -1,21 +1,8 @@
-world_name: empty
+include: world_model_gmapping_sim.yaml
```
Comment (jlunenburg) [inline parameters/world_modeling/world_model_gmapping.yaml]: It seems really odd to me to include sim stuff in a non-sim file

Comment (MatthijsBurgh) [reply parameters/world_modeling/world_model_gmapping.yaml]: Yep, but the robot config is an extension of the sim config. (Difference most of the times is the rgbd obstacle avoidance.)

**Pull request merged**